project01/artur/direction.h

```diff
@@ -0,0 +1,7 @@
+enum Direction
```

Comment:
Nice that you added this to a different file, although you should add `#pragma once`, otherwise you might get multiple definitions - ODR violation :skull: